installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -6,20 +6,25 @@ include::_attributes/common-attributes.adoc[]
+You can install an {product-title} cluster on vSphere using a variety of different installation methods.
+Each method has qualities that can make them more suitable for different use cases, such as installing a cluster in a disconnected environment or installing a cluster with minimal configuration and provisioning.
 
-You can also install {product-title} on vSphere by using installer-provisioned or user-provisioned infrastructure. Installer-provisioned infrastructure is ideal for installing in environments with air-gapped/restricted networks, where the installation program provisions the underlying infrastructure for the cluster. You can also install {product-title} on infrastructure that you provide. If you do not use infrastructure that the installation program provisions, you must manage and maintain the cluster resources yourself.
+== Assisted Installer
 
-See the xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
+You can install {product-title} with the link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform[{ai-full}]. This method requires no setup for the installer, and is ideal for connected environments like vSphere. Installing with the {ai-full} also provides integration with vSphere, enabling autoscaling. See xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc#installing-on-prem-assisted[Installing an on-premise cluster using the {ai-full}] for additional details.
 
-[IMPORTANT]
-====
-The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the vSphere platform and the installation process of {product-title}. Use the user-provisioned infrastructure installation instructions as a guide; you are free to create the required resources through other methods.
-====
+== Agent-based Installer
+
+You can install an {product-title} cluster on vSphere using the Agent-based Installer.
+The Agent-based Installer can be used to boot an on-premises server in a disconnected environment using a bootable image.
+With the Agent-based Installer, users also have the flexibility to provision infrastructure, customize network configurations, and customize installations within a disconnected environment.
+See xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Preparing to install with the Agent-based Installer] for additional details.
```

Comment:
The guidance is not clear but would adding the context variable to each xref make for better xref consistency?

xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer_preparing-to-install-with-agent-based-installer[Preparing to install with the Agent-based Installer]

Reply:
I'm honestly not sure. I think the worst case is that, as with any other time you get the anchor ID of an xref incorrect, the build doesn't break and the link takes you to the top of the assembly being linked to. But in that case it would be like unintentionally producing the right effect while still doing something the wrong way.

I think in our [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-internal-content) the examples state to use `#anchor-id`, in which case the accurate ID for the top level is just `preparing-to-install-with-agent-based-installer`, so I think I should keep this as is.